spec/javascripts/packages/document-capture/context/acuant-sdk-upgrade-a-b-test-spec.jsx

```diff
@@ -0,0 +1,120 @@
+import { useContext } from 'react';
```

Comment:
Since this is a new file and [all new code should be TypeScript](https://github.com/18F/identity-idp/blob/main/docs/frontend.md#javascript), can we make this file `.tsx`?